roles/pulp3/tasks/install.yml

```diff
@@ -99,5 +99,12 @@
       with_dict: '{{ pulp_install_plugins }}'
       when: pulp_install_plugins[item.key].source_dir is defined
 
+    - name: Install requirements for building docs
```

Comment:
This is supposed to go into the pulp3-devel role. Otherwise these deps are going to be installed for users that are just installing pulp.

Reply:
Oops, good catch.